ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayKml/DisplayKml.qml

```diff
@@ -0,0 +1,145 @@
+// [WriteFile Name=DisplayKml, Category=Layers]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Window 2.3
+import Esri.ArcGISRuntime 100.4
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: (Screen.logicalPixelDensity * 25.4) / (Qt.platform.os === "windows" || Qt.platform.os === "linux" ? 96 : 72)
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        Scene {
+            id: scene
+            BasemapImageryWithLabels{}
+
+            Surface {
+                ArcGISTiledElevationSource {
+                    url: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+            }
+        }
+    }
+
+    ViewpointCenter {
+        id: viewpoint
+
+        targetScale: 10000000
+        center: Point {
+            x: -98
+            y: 39
+            spatialReference: SpatialReference {
+                wkid: 4326
+            }
+        }
+
+        Camera {
+            distance: viewpoint.targetScale
+            heading: 0
+            pitch: 0
+            roll: 0
+            location: viewpoint.center
+        }
+    }
+
+    ComboBox {
+        anchors {
+            right: parent.right
+            top: parent.top
+            margins: 5 * scaleFactor
+        }
+        width: 100 * scaleFactor
+
+        model: ["URL", "Local file", "Portal Item"]
+
+        onCurrentIndexChanged: {
+            // clear previous layers
+            scene.operationalLayers.clear();
+
+            // create KML variable
+            var kmlLayer;
```

Comment:
Don't actually need to declare this here. Javascript only brace-scopes fo functions.

```
            if (currentIndex === 0)
                var kmlLayer = createFromUrl();
            else if (currentIndex === 1)
                var kmlLayer = createFromFile();
            else
                var kmlLayer = createFromPortalItem();
```

Is valid.

Reply:
it's valid, but I get a warning in Qt Creator that says `kmlLayer is declared more than once`